scrunch/datasets.py

```diff
@@ -2495,3 +2493,18 @@ def transformations(self):
         if 'transform' in self.view:
             return self.view.transform.insertions
         return None
+
+    def edit_resolution(self, resolution):
+        """
+        PATCHes the rollup_resolution attribute of a datetime variable. This is the
+        equivalent to the UI's change resolution action. 
+
+        Be sure to grab the editor's lock of the dataset.
+        
+        :usage: edited_var = var.edit_resolution('M')
+        assert editar_var.rollup_resolution == 'M'
+        """
+        assert self.type == 'datetime', 'Method only allowed for datetime variables'
+        self.dataset._validate_vartypes(self.type, resolution=resolution)
+        self.resource.edit(view=dict(rollup_resolution=resolution))
```

Comment:
This is wrong, note that here you will overwrite all the other attributes in the variable's view attribute. You need to read the current dict, then change the value an send that as the new dict. Sending a new dict will overwrite the full thing.